utils/build/virtual_machine/weblogs/ruby/provision_test-app-ruby-multialpine.yml

```diff
@@ -0,0 +1,25 @@
+weblog:
+    name: test-app-ruby-multialpine
+    exact_os_branches: [ubuntu24]
```

Comment:
We'll set this by command line. We should remove this from here, because actually it should work on all vms